Add project organization and roles (#74)
PrestaShop/open-source: content/maintainers-guide/project-organization.md

### Maintainer

Maintainers provide the project's technical vision, review code contributions, and ensure that the software develops in line with the product vision.

Current maintainers:

- Franck Lefèvre ([@Progi1984](https://github.com/Progi1984))
- Ibrahima Sow ([@sowbiba](https://github.com/sowbiba))
- Jonathan Lelièvre ([@jolelievre](https://github.com/jolelievre))
- Krystian Podemski ([@kpodemski](https://github.com/kpodemski))
- Mathieu Ferment ([@matks](https://github.com/matks))
- Matthieu Rolland ([@matthieu-rolland](https://github.com/matthieu-rolland))
- Pablo Borowicz ([@eternoendless](https://github.com/eternoendless))
- Pierre Rambaud ([@PierreRambaud](https://github.com/PierreRambaud))
- Pululu Kinanga André ([@PululuK](https://github.com/PululuK))
- Thomas Baccelli ([@atomiix](https://github.com/atomiix))
- Valentin Szczupak ([@NeOMakinG](https://github.com/NeOMakinG))

### UX Designer

UX Designers make sure that interfaces follow the best practices in usability and design.

Current UX designers:

- Scott Cosio ([@prestascott](https://github.com/prestascott))
- Tristan Lehot ([@TristanLDD](https://github.com/TristanLDD))

> ### Quality Assurance (QA)

Reviewer: I think you forgot to delete this.
```suggestion
```

> 
> Issue managers qualify new issues to avoid duplicates and, if they are indeed defects in the software, their severity.
> 
> Current issue managers:

Reviewer: ```suggestion
### Issue Managers

Issue managers reproduce and qualify new issues to avoid duplicates and, if they are indeed defects in the software, their severity. They also redirect new improvement suggestions to the right person in the PM team.

Current issue managers:
```

- Florine Hea ([@florine2623](https://github.com/florine2623))
- Hana Rebai ([@HanaRebaiQA](https://github.com/HanaRebaiQA))
- Hibatallah Aouadni ([@hibatallahAouadni](https://github.com/hibatallahAouadni))
- Khouloud Belguith ([@khouloudbelguith](https://github.com/khouloudbelguith))
- Marwa Chelly ([@marwachelly](https://github.com/marwachelly))

> QA analysts make sure that the quality of the software improves over time, by verifying that code changes behave as expected.
> 
> Current QA analysts:

Reviewer: ```suggestion
### Software Testers

Software Testers make sure that the quality of the software improves over time, by verifying that code changes behave as expected. To do this, they extensively test Pull Requests, PrestaShop's releases and its natives modules.

Current Software Testers:
```

---
title: Project organization
weight: 1
---

# Project organization

## Project roles

> ### Product Manager

Reviewer 2: ```suggestion
### Product Managers
```

Product Managers provide the project's product vision, ensuring that the software provides an answer to the needs of its different types of users.

Current product managers:

> - Fabien Vallon ([@fabienvallon](https://github.com/fabienvallon))

Reviewer 3: Maybe we could clarify that @fabienvallon is the lead PM ?

Reviewer 4: I disagree. Why ? In the open source project, PM have the same status like maintainers. No lead. Like @eternoendless in maintainers group.

Author: Same. Until we determine a proper role for Fabien and me, officially we're equal.

Reviewer 5: I agree Progi1984.

> - Marion François ([@marionf](https://github.com/marionf))

Reviewer 6: ```suggestion
```

- Mateus Shirlaw ([@MatShir](https://github.com/MatShir))

> ### Maintainer

Reviewer 7: I think it's about the `Maintainer` role, not about every maintainers, same for every other roles

Maintainers provide the project's technical vision, review code contributions, and ensure that the software develops in line with the product vision.

Current maintainers:

- Franck Lefèvre ([@Progi1984](https://github.com/Progi1984))
- Ibrahima Sow ([@sowbiba](https://github.com/sowbiba))
- Jonathan Lelièvre ([@jolelievre](https://github.com/jolelievre))
- Krystian Podemski ([@kpodemski](https://github.com/kpodemski))
- Mathieu Ferment ([@matks](https://github.com/matks))
- Matthieu Rolland ([@matthieu-rolland](https://github.com/matthieu-rolland))
- Pablo Borowicz ([@eternoendless](https://github.com/eternoendless))
- Pierre Rambaud ([@PierreRambaud](https://github.com/PierreRambaud))
- Pululu Kinanga André ([@PululuK](https://github.com/PululuK))
- Thomas Baccelli ([@atomiix](https://github.com/atomiix))
- Valentin Szczupak ([@NeOMakinG](https://github.com/NeOMakinG))

> ### UX Designer

Reviewer 3: ```suggestion
### Product Designer
```

Reviewer 3: @TristanLDD @prestascott we rather speak of product designer ?

- Khouloud Belguith ([@khouloudbelguith](https://github.com/khouloudbelguith))
- Marwa Chelly ([@marwachelly](https://github.com/marwachelly))

QA analysts make sure that the quality of the software improves over time, by verifying that code changes behave as expected.

Current QA analysts:

- Florine Hea ([@florine2623](https://github.com/florine2623))
- Hana Rebai ([@HanaRebaiQA](https://github.com/HanaRebaiQA))
- Hibatallah Aouadni ([@hibatallahAouadni](https://github.com/hibatallahAouadni))
- Khouloud Belguith ([@khouloudbelguith](https://github.com/khouloudbelguith))
- Marwa Chelly ([@marwachelly](https://github.com/marwachelly))
- Robin Fischer ([@Robin-Fischer-PS](https://github.com/Robin-Fischer-PS))
- Sarah Dib ([@sarahdib](https://github.com/sarahdib))

QA automation engineers write automated functional tests, which help accelerate the work of QA analysts.

Current QA automation engineers:

- Nesrine Abdmouleh ([@nesrineabdmouleh](https://github.com/nesrineabdmouleh))
- Stéphane Decisy ([@SD1982](https://github.com/SD1982))
- Boubker Bribri ([@boubkerbribri](https://github.com/boubkerbribri))

> ### Wording Manager

Reviewer 3: ```suggestion
### Content Manager
```

Reviewer 3: Ping @Julievrz

> ### Wording Manager

Reviewer 2: ```suggestion
### Product Content Manager
```

That's the full name 😇

Reviewer 5: Sound pompous for an Open Source project to me.
But if you want to keep 'Product Content Manager' , it's fine

Reviewer 2: "Content Manager" would work too, but "wording manager" sounds a bit simplistic to me. 😇

Reviewer 5: This is role, not job.
Your job is more complexe than the role

For your job ( Product Content Manager ) ,  2 roles for the community 
- Wording  maintener 
- Documentation maintener

Author: Can we please settle on a name for this role? We can add more detail after if necessary.

> Wording managers ensure that wordings are adequate, follow the project's tone of voice, and are categorized properly.

Reviewer 3: ```suggestion
Content managers ensure that wordings are adequate, follow the project's tone of voice, and are categorized properly.
```

> Wording managers ensure that wordings are adequate, follow the project's tone of voice, and are categorized properly.

Reviewer 2: ```suggestion
Product Content Managers are responsible for improving the user experience by ensuring the quality and consistency of software content and user documentation.
```

Author: I chose "wording" because I didn't think of documentation. Documentation management is a different role that could be done by different contributors. So you're actually doing two roles :P

How about I add a specific role for that?

Reviewer 2: Sounds good ;-)

Author: Alright!

> Current wording managers:

Reviewer 3: ```suggestion
Current content managers:
```

> Current wording managers:

Reviewer 2: ```suggestion
Current Product Content Managers:
```
